bot/controllers/viewPoints.js

```diff
@@ -0,0 +1,38 @@
+const BaseController = require('../baseController.js');
+const Command = require('../baseCommand.js');
+const {
+  Member,
+} = require('../../db/models');
+
+class ViewPointsController extends BaseController {
+  constructor(message) {
+    super(message);
+    const controller = this;
+    this.commands = [
+      new Command(
+        '!viewPoints',
+        '!viewPoints',
+        'Returns current point total.',
+        'Receive a message with your current point total.',
+        this.viewPointsAction.bind(controller),
+        'dm',
+      ),
+    ];
+  }
+
+  async viewPointsAction() {
+    const { message } = this;
+    // Get Member Data
+    const memberData = await Member.findAll({
```

Comment:
Not sure if it would make a huge difference or not, but if you're just getting one user there is `Member.findOne` that can be used. It helped fix some of my problems oddly enough, but just a thought. I'm going to change my util functions to that though.